packages/geo/src/lib/query/shared/query.directive.ts

```diff
@@ -198,6 +205,21 @@ export class QueryDirective implements AfterViewInit, OnDestroy {
         event.pixel,
         (featureOL: OlFeature<OlGeometry>, layerOL: any) => {
           const layer = this.map.getLayerById(layerOL.values_._layer.id);
+          // check no query param workspace
+          if (layer.options.workspace && layer.options.workspace.noMapQueryOnOpenTab) {
+            if(this.queryService.workspace &&
```

Comment:
Will it work if you only check if the workspace defined in the layer options is active ? 
Something like: 
if (layer.options.workspace?.active && layer.options.workspace.noMapQueryOnOpenTab) {
return;
}

Reply:
the queryService.workspace is important for case :  workspace tab is open but no layer is select.
In that case workspace is active and queryService.workspace = undefined.

Reply:
In that case, i think we should rewrite the active attribute @pelord ? 'active' is a workspace attribute and should be true only if the workspace is the selectedWorkspace ? Use the workspaceState to adjust the current workspace 'active' attribute